CHANGELOG.md

```diff
@@ -85,6 +85,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed [#1937](https://github.com/JabRef/jabref/issues/1937): If no help page for the current chosen language exists, the english help page will be shown
 - Fixed [#2060](https://github.com/JabRef/jabref/issues/2060): Medline fetcher now imports data in UTF-8 encoding
 - Fixed file menu displays wrong hotkey in the German translation
+- Fixed [#2090](https://github.com/JabRef/jabref/issues/#2090): Two separator glitch in menu edit 
```

Comment:
I would reformulate/explain that a bit more in the changelog a bit, saying sth like:  that if the special fields were not selected than the two menu item separator where shown
Apart from that, it looks good :+1: